UnitsNet.Tests/CustomCode/PressureTests.cs

```diff
@@ -117,5 +120,102 @@ public void PressureDividedByLengthEqualsSpecificWeight()
             SpecificWeight specificWeight = Pressure.FromPascals(20) / Length.FromMeters(2);
             Assert.Equal(SpecificWeight.FromNewtonsPerCubicMeter(10), specificWeight);
         }
+        // Pressure Measurement References
+        [Fact]
+        public void ReferenceConversion_WithDefaultPressureReference_PressureReferenceIsAbsolute()
+        {
+            ReferencePressure refPressure = new ReferencePressure(Pressure.FromAtmospheres(3));
+            AssertEx.Equals(PressureReference.Absolute, refPressure.Reference);
+        }
+        [Fact]
```

Comment:
Minor, but if you sorted these methods alphabetically then it would be easier to read and verify that you are verifying all three properties Absolute, Gauge and Vacuum for each of the the different reference pressures.